docs/kit/Adoption View/00_adoption-view.md

```diff
@@ -1,27 +1,58 @@
-# Adoption View
+---
+id: Adoption View
+title: Adoption View
+description: 'Connector Kit'
+sidebar_position: 1
+---
 
-The ConnectorKit provides a connector framework, based on the [Eclipse Dataspace Connector][edc-url] for sovereign, cross-enterprise data exchange.
+## Introduction
+
+The ConnectorKit provides a connector framework, based on the [Eclipse Dataspace Components Connector][edc-url] for sovereign, cross-enterprise data exchange.
+
+## Vision & Mission
+
+### Vision
+
+Sharing data along the automotive supply chain adds great value for every participant. The key to such collaboration is trust in the involved partners and assurance that the shared data is not being misused. Within a few years, companies along the automotive value chain will share and exchange data with each other as a matter of course, without anyone fearing that their data will be misused while unlocking significant values for every party.
+
+### Mission
+
+Collaboration needs trust. Within Catena-X trust is built on an open-source architecture and components designed to support full sovereignity.
+The Connector KIT provides the single source of information and technology to perform trustful data exchange within Catena-X. The Connector KIT bundles the interaction patterns, relevant standards, APIs, and reference implementations for developers.
+
+## Concept
+
+Trust, interoperability, and data sovereignty are the objectives and values for secure and sustainable peer-to-peer data exchange between organizations and companies. The claim is data sovereignty: Whoever provides data to the Data Space retains control and decides individually who is involved in the data exchange, how, when, where and under what conditions. A corresponding concept was developed in the context of [Gaia-X][gaiax-url] and the [International Data Space Association][idsa-url]. __The essential software component is the EDC Connector.__
```

Comment:
The C in EDC is short for Connector. So I'd prefer to call it the EDC or the Eclipse Dataspace Connector.

Reply:
Comes up a few times in this file.